### PR TITLE
Ingestion: eacl 2023 workshops

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

**Comment by anthology-assist** [comment]
This can be merged. Waiting on 3 more workshops.

File: data/yaml/venues/unlp.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+acronym: UNLP
+name: Workshop on Ukrainian Natural Language Processing (UNLP 2023)
```
Comment (mjpost) [inline data/yaml/venues/unlp.yaml:2]: In the venue entry, there shouldn't be a year

File: data/yaml/venues/c3nlp.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+acronym: C3NLP
+name: Cross-Cultural Considerations in NLP @ EACL
```
Comment (mjpost) [inline data/yaml/venues/c3nlp.yaml:2]: The venue entry shouldn't mention a particular parent conference (remove `@ EACL`)

**Review by mjpost — COMMENTED** [review]
Please note a few small comments in the venue files. Then we can merge.

File: data/yaml/venues/unlp.yaml (hunk 1 above)
Comment (mbollmann) [reply data/yaml/venues/unlp.yaml:2]: Also no acronym in the name (we show that from the acronym field already)

**Review by mjpost — APPROVED** [review]

**Pull request merged**